src/settypes/recursiveset.jl

```diff
@@ -37,7 +37,11 @@ function print_recursiveset(io::IO, rs::RecursiveSet{T}; offset) where {T}
     end
 end
 
-function Base.show(io::IO, rs::RecursiveSet{T}) where {T}
+function Base.show(io::IO, rs::RecursiveSet)
+    return print_recursiveset(io, rs; offset=0)
+end
+
+function Base.show(io::IO, ::MIME"text/plain", rs::RecursiveSet)
     return print_recursiveset(io, rs; offset=0)
 end
```

Comment:
Isn't the second falling back on the first?

Reply:
I removed it in the latest commit and I get this:
```julia
julia> RecursiveSet{Int}(1)
Error showing value of type RecursiveSet{Int64}:
ERROR: MethodError: no method matching length(::RecursiveSet{Int64})

Closest candidates are:
  length(::LibGit2.GitBlob)
   @ LibGit2 ~/.julia/juliaup/julia-1.10.3+0.x64.linux.gnu/share/julia/stdlib/v1.10/LibGit2/src/blob.jl:3
  length(::Base.EnvDict)
   @ Base env.jl:224
  length(::VSCodeServer.JSON.Parser.MemoryParserState)
   @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.79.2/scripts/packages/JSON/src/Parser.jl:29
  ...

Stacktrace:
  [1] summary(io::IOContext{Base.TTY}, t::RecursiveSet{Int64})
    @ Base ./show.jl:213
  [2] show(io::IOContext{Base.TTY}, ::MIME{Symbol("text/plain")}, t::RecursiveSet{Int64})
    @ Base ./show.jl:224
  [3] (::OhMyREPL.var"#15#16"{REPL.REPLDisplay{…}, MIME{…}, Base.RefValue{…}})(io::IOContext{Base.TTY})
    @ OhMyREPL ~/.julia/packages/OhMyREPL/6UG7a/src/output_prompt_overwrite.jl:23
  [4] with_repl_linfo(f::Any, repl::REPL.LineEditREPL)
    @ REPL ~/.julia/juliaup/julia-1.10.3+0.x64.linux.gnu/share/julia/stdlib/v1.10/REPL/src/REPL.jl:569
  [5] display
    @ ~/.julia/packages/OhMyREPL/6UG7a/src/output_prompt_overwrite.jl:6 [inlined]
  [6] display(d::REPL.REPLDisplay, x::Any)
    @ REPL ~/.julia/juliaup/julia-1.10.3+0.x64.linux.gnu/share/julia/stdlib/v1.10/REPL/src/REPL.jl:278
  [7] display(x::Any)
    @ Base.Multimedia ./multimedia.jl:340
  [8] #invokelatest#2
    @ ./essentials.jl:892 [inlined]
  [9] invokelatest
    @ ./essentials.jl:889 [inlined]
 [10] print_response(errio::IO, response::Any, show_value::Bool, have_color::Bool, specialdisplay::Union{…})
    @ REPL ~/.julia/juliaup/julia-1.10.3+0.x64.linux.gnu/share/julia/stdlib/v1.10/REPL/src/REPL.jl:315
 [11] (::REPL.var"#57#58"{REPL.LineEditREPL, Pair{Any, Bool}, Bool, Bool})(io::Any)
    @ REPL ~/.julia/juliaup/julia-1.10.3+0.x64.linux.gnu/share/julia/stdlib/v1.10/REPL/src/REPL.jl:284
 [12] with_repl_linfo(f::Any, repl::REPL.LineEditREPL)
    @ REPL ~/.julia/juliaup/julia-1.10.3+0.x64.linux.gnu/share/julia/stdlib/v1.10/REPL/src/REPL.jl:569
 [13] print_response(repl::REPL.AbstractREPL, response::Any, show_value::Bool, have_color::Bool)
    @ REPL ~/.julia/juliaup/julia-1.10.3+0.x64.linux.gnu/share/julia/stdlib/v1.10/REPL/src/REPL.jl:282
 [14] (::REPL.var"#do_respond#80"{…})(s::REPL.LineEdit.MIState, buf::Any, ok::Bool)
    @ REPL ~/.julia/juliaup/julia-1.10.3+0.x64.linux.gnu/share/julia/stdlib/v1.10/REPL/src/REPL.jl:911
 [15] (::VSCodeServer.var"#103#106"{REPL.var"#do_respond#80"{…}})(mi::REPL.LineEdit.MIState, buf::IOBuffer, ok::Bool)
    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.79.2/scripts/packages/VSCodeServer/src/repl.jl:122
 [16] #invokelatest#2
    @ ./essentials.jl:892 [inlined]
 [17] invokelatest
    @ ./essentials.jl:889 [inlined]
 [18] run_interface(terminal::REPL.Terminals.TextTerminal, m::REPL.LineEdit.ModalInterface, s::REPL.LineEdit.MIState)
    @ REPL.LineEdit ~/.julia/juliaup/julia-1.10.3+0.x64.linux.gnu/share/julia/stdlib/v1.10/REPL/src/LineEdit.jl:2656
 [19] run_frontend(repl::REPL.LineEditREPL, backend::REPL.REPLBackendRef)
    @ REPL ~/.julia/juliaup/julia-1.10.3+0.x64.linux.gnu/share/julia/stdlib/v1.10/REPL/src/REPL.jl:1312
 [20] (::REPL.var"#62#68"{REPL.LineEditREPL, REPL.REPLBackendRef})()
    @ REPL ~/.julia/juliaup/julia-1.10.3+0.x64.linux.gnu/share/julia/stdlib/v1.10/REPL/src/REPL.jl:386
Some type information was truncated. Use `show(err)` to see complete types.
```